src/include/sof/audio/dcblock/dcblock.h

```diff
@@ -11,6 +11,18 @@
 #include <stdint.h>
 #include <sof/platform.h>
 #include <ipc/stream.h>
+#include <sof/compiler_info.h>
+
+/* __XCC__ is both for xt_xcc and xt_clang */
+#if defined(__XCC__)
```

Comment:
Looks like you dropped `XCHAL_HAVE_HIFI4` which was in the previous revision.

https://sof-ci.01.org/sofpr/PR7214/build4384/build/tgl_zph_error.txt

```
sof/src/audio/dcblock/dcblock_hifi4.c:14:33: error: 
xtensa/tie/xt_hifi4.h: No such file or directory
```